transform/snowflake-dbt/models/hightouch/product_campaigns/cloud_signup_campaign_facts.sql

```diff
@@ -92,11 +96,15 @@ select
     customer_facts.trial_end,
     customer_facts.company_name,
     customer_facts.email,
+    coalesce(customer_facts.installation_state, 'hibernating') as installation_state,
     server_facts.last_active_date,
     server_facts.cloud_posts_total,
     server_facts.cloud_mau,
     server_facts.cloud_dau,
-    server_facts.cloud_posts_daily
+    server_facts.cloud_posts_daily,
+    -- server_facts.total_storage, 
```

Comment:
nit: remove commented out code

Reply:
total_storage will be added here once raw data is available.